dockerfiles/rsyslog/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.4
+
+RUN  apk add --update "rsyslog=8.18.0-r0" \
+  && rm -rf /var/cache/apk/*
+
+EXPOSE 514 514/udp
+
+VOLUME [ "/var/log", "/etc/rsyslog.d" ]
```

Comment:
it may be needed for mounting extra rsyslog configs, today we don't have any but we may have some in future.